### PR TITLE
[CIR-422] Added PayTaxInUK page

### PR DESCRIPTION
**Pull request by MJCallahanPage**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -290,6 +290,15 @@ deemedParent.checkYourAnswersLabel = Does the group have a 'deemed' parent compa
 deemedParent.error.required = Select yes if the group has a 'deemed' parent company?
 
 
+# PayTaxInUkPage Messages
+# ----------------------------------------------------------
+payTaxInUk.title = Does Name have a Unique Taxpayer Reference?
+payTaxInUk.heading = Does Name have a Unique Taxpayer Reference?
+payTaxInUk.hint = This could be for paying Corporation Tax or completing Self Assessment in the UK.
+payTaxInUk.checkYourAnswersLabel = Does Name have a Unique Taxpayer Reference?
+payTaxInUk.error.required = Select yes if Does Name have a Unique Taxpayer Reference?
```
Comment (LadonJackson) [inline conf/messages.cy]: I believe this should match the messages.en

File: app/controllers/groupStructure/PayTaxInUkController.scala [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import controllers.BaseController
+import controllers.actions._
+import forms.PayTaxInUkFormProvider
+import handlers.ErrorHandler
+import javax.inject.Inject
+import models.Mode
+import models.requests.DataRequest
+import navigation.GroupStructureNavigator
+import pages.groupStructure.{ParentCompanyNamePage, PayTaxInUkPage}
+import play.api.data.Form
```
Comment (LadonJackson) [inline app/controllers/groupStructure/PayTaxInUkController.scala]: unused import

File: test/views/groupStructure/PayTaxInUkViewSpec.scala [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+package views.groupStructure
+
+import assets.messages.SectionHeaderMessages
+import assets.messages.groupStructure.PayTaxInUkMessages
+import controllers.groupStructure.routes
+import forms.PayTaxInUkFormProvider
+import models.NormalMode
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import views.{BaseSelectors, Twirl}
```
Comment (LadonJackson) [inline test/views/groupStructure/PayTaxInUkViewSpec.scala]: unused import

File: test/views/groupStructure/PayTaxInUkViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      messageKeyPrefix,
+      expectedFormAction = routes.PayTaxInUkController.onSubmit(NormalMode).url,
+      headingArgs = Seq(companyName),
+      section = section
+    )
+
+    behave like pageWithSubHeading(applyView(form), SectionHeaderMessages.groupStructure)
+
+    behave like pageWithSaveForLater(applyView(form))
+
```
Comment (peckoverHMRC) [inline test/views/groupStructure/PayTaxInUkViewSpec.scala]: More tests please

**Pull request merged**